src/control/Edit/Edit.js

```diff
@@ -1,6 +1,5 @@
 import Control from 'ol/control/Control';
 import { CLASS_CONTROL, CLASS_UNSELECTABLE } from 'ol/css';
-import EventType from 'ol/events/EventType';
```

Comment:
`ol/events/EventType` isn't actually part of the public OpenLayers API and thus it breaks when trying to access `ol.events.EventType` - see https://openlayers.org/en/latest/apidoc/module-ol_events.html

Also, even if it were available, this usage is a little weird. This control isn't using these constants for consuming OpenLayers events, only standard browser events - which are typically consumed with a hard-coded string because (for whatever reason) browser makers didn't see fit to expose a constant for it. :(